cmds/server.go

```diff
@@ -11,11 +11,13 @@ and nonsense.
 `
 
 var listenOn string
+var logRequests bool
 
 const defaultListenOn = "127.0.0.1:8297"
 
 func init() {
 	serverCmd.Flags().StringVarP(&listenOn, "listen", "l", defaultListenOn, "IP address and port to listen on")
+	serverCmd.Flags().BoolVarP(&logRequests, "log-requests", "", false, "Log requests to stderr")
```

Comment:
Should we just go ahead and always log? I just didn't want to solve this because I was lazy but I'm not sure it's inappropriate to log all the time. But second question - should we log to STDOUT, since that's a 12-factor standard?

Reply:
i was thinking about doing just that, but then wasn't sure what to do with the logs when we run via launchd. do you know how to set that up?

Reply:
and yes, we can change to stdout to follow 12-factor.

Reply:
Not exactly, but I'm OK with dropping them on the floor for starters and iterating toward doing it better over time. The real value so far has proven to be interactive testing anyway.